debezium-core/src/test/java/io/debezium/kafka/KafkaClusterTest.java

```diff
@@ -195,6 +198,32 @@ public void shouldStartClusterAndAllowAsynchronousProductionAndAutomaticConsumer
         assertThat(messagesRead.get()).isEqualTo(numMessages);
     }
 
+    @Test
+    public void shouldSetClusterConfigProperty() throws Exception {
+        Properties config = new Properties();
+        config.put("foo", "bar");
+        KafkaCluster kafkaCluster = new KafkaCluster().withKafkaConfiguration(config);
+
+        Field kafkaConfigField = KafkaCluster.class.getDeclaredField("kafkaConfig");
+        kafkaConfigField.setAccessible(true);
+        Properties kafkaConfig = (Properties) kafkaConfigField.get(kafkaCluster);
+        assertThat(kafkaConfig).hasSize(1);
+    }
+
+    @Test
+    public void shouldSetServerConfigProperty() throws Exception {
+        Properties config = new Properties();
+        config.put("foo", "bar");
+        KafkaCluster kafkaCluster = new KafkaCluster().withKafkaConfiguration(config).addBrokers(1);
+
+        Field kafkaServersField = KafkaCluster.class.getDeclaredField("kafkaServers");
+        kafkaServersField.setAccessible(true);
```

Comment:
Would this be compilable under Java 9?

Reply:
It is compileable and also runnable on Java 9. For now due to the fact that `KafkaCluster` is part of the unnamed module when this test is run. But also assuming that _debezium-core_ will be a named module eventually, it'll be fine as then the unit tests in all likelyhood will be patched into that method for execution, and `setAccessible()` is always allowed if caller and destination are in the same module.